### PR TITLE
Moved the scripting logic back into the Makefile and separated app venv from ci venv

### PR DESCRIPTION
**Pull request by TyMarc**
Also deleted the stubs and python packaging dependencies for the application

### CODE REVIEW

File: flare_splunk_integration/flare_splunk_integration [hunk 1]
```diff
@@ -0,0 +1 @@
+/Users/marc-antoinehinse/repos/splunk-integration/flare_splunk_integration
```
Comment (markkasaboski) [inline flare_splunk_integration/flare_splunk_integration]: What's this?

File: requirements.tools.txt [hunk 2]
```diff
@@ -1,3 +1,4 @@
+pytest==8.3.2
```
Comment (markkasaboski) [inline requirements.tools.txt:1]: Oh! Nice catch. Missed that.

File: flare_splunk_integration/flare_splunk_integration (hunk 1 above)
Comment (TyMarc) [reply flare_splunk_integration/flare_splunk_integration]: Oh wow good catch

**Review by markkasaboski — APPROVED** [review]
Just that one file I'm not sure about. Otherwise, LGTM

**Pull request merged**